### DISCUSSION TITLE
Summary/Revision and Quiz

### DISCUSSION THREAD

**Discussion by Mims9141** [TheOdinProject/curriculum#26327] [Ideas]
I have been learning the foundations path  and I thought it would be great if there was summary/revision of each course. Especially as I have taken a long break, a nice summary to go over would be great to refresh the memory rather than going over everything again. Also maybe a nice quiz feature for those who haven't been on the site in a while or as you progress, a short pop up quiz on what you have learnt. 

It can get frustrating when taking long breaks, feels like a have regressed soo much, even though i probably have the information somewhere at the back of my mind. As we go through the course, we may also forget many of the things along the way so maybe this suggestion can help mitigate that.

What do you guys think?

**Comment by CouchofTomato** [comment]
Thanks for your idea @Mims9141 

How do you see the summary being presented?

**Reply by Mims9141** [reply]
I envision the summary to be conveniently placed at the end of each course, designed to be concise and straight to the point. It should primarily highlight the major key points and, where relevant, include code examples to aid in building muscle memory. Additionally, a set of a few targeted questions at the end of the summary could help encapsulate the core concepts learned throughout the course.
